app/controllers/skills/new.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.ObjectController.extend({
+  needs: ['skills'],
+  // replace this with data from rails api call (get all groups)
+  groups: ["1407", "1409", "1410", "1412"],
```

Comment:
How are we dealing with "instructors" ? Originally, this was going to be just another group. Did we add an instructor column to the user table? Are we handling instructors more like a traditional admin?